resources/lib/widgets.py

```diff
@@ -379,18 +392,30 @@ def get_widget_content(handle, params):
         log.debug("Recommended Items : {0}".format(len(ids), id_list))
         url_params["Ids"] = id_list
 
+    # items_url = get_jellyfin_url(url_verb, url_params)
+    #
+    # list_items, detected_type, total_records = process_directory(items_url, None, params, use_cached_widget_data)
+
+    # remove resumable items from next up
+    # if widget_type == "nextup_episodes":
+    #     filtered_list = []
+    #     for item in list_items:
+    #         resume_time = item[1].getProperty("ResumeTime")
+    #         if resume_time is None or float(resume_time) == 0.0:
+    #             filtered_list.append(item)
+    #     list_items = filtered_list
+
     items_url = get_jellyfin_url(url_verb, url_params)
+    items_url2 = get_jellyfin_url(url_verb2, url_params2)
 
     list_items, detected_type, total_records = process_directory(items_url, None, params, use_cached_widget_data)
+    list_items2, detected_type, total_records = process_directory(items_url2, None, params, use_cached_widget_data)
 
     # remove resumable items from next up
     if widget_type == "nextup_episodes":
-        filtered_list = []
-        for item in list_items:
-            resume_time = item[1].getProperty("ResumeTime")
-            if resume_time is None or float(resume_time) == 0.0:
-                filtered_list.append(item)
-        list_items = filtered_list
+    #xbmcplugin.addSortMethod(handle, xbmcplugin.SORT_METHOD_LASTPLAYED)
+        for item in list_items2:
+            list_items.append(item)
```

Comment:
Since this was moved higher into the logic block, (and it appears that it didn't work before anyway, probably because of server changes), we can remove this entire block now, from the `# remove resumable items` comment to the start of `if detected_type is not None`